src/plugins/no_data_page/common/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * @internal
+ *
+ * TODO: consider moving this to a package under the
+ * Security domain as it is tied to a Security API
+ * endpoint.
+ */
+export const NO_DATA_API_PATHS: Record<string, Record<string, string>> = {
+  internal: {
+    /**
+     * Use the Security plugin's endpoint
+     * to determine if the user has active API Keys
+     */
+    hasApiKeys: '/internal/security/api_key/_has_active',
```

Comment:
Related discussion, about parts of code added in this PR that are closely binded to the Security domain: https://github.com/elastic/kibana/pull/172225#discussion_r1432307325